COMPLETE LIST OF SCRIPTS.vbs

```diff
@@ -717,7 +717,7 @@ script_array(script_num).subcategory            = array("")  '<<Temporarily remo
 script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("CM Emergency_Aid_Eligibility_-_SNAP/Expedited_Food 04.04","CM Applications 05","CM Application_-_Pending Cases 05.09.12","TE Notice_of_Interview/Missed_Interview_(NOMI) 02.05.15","TE Expedited_Food_Support_Tip_Sheet 13.031", "SHAREPOINT Applications https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Applications.aspx")						'SEE Line 58 for format')						'SEE Line 58 for format'
```

Comment:
"CM Application_-_Pending Cases 05.09.12"give me an error in the browser: access to dhs.state.mn.us was denied. However I am able to pull up 05.09.12 on a separate tab with no issues. Not sure if this is a me issue or if the "-" in the name is possibly problematic?

Reply:
This is not linking correctly, but it looks like it is missing an underscore between Pending and Cases. It should be "CM Application_-_Pending_Cases 05.09.12"